samples/grids/grid/editing-excel-style/src/index.tsx

```diff
@@ -1,146 +1,138 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
```

Comment:
The `React` import is unused and should be removed

Reply:
while correct, current eslint throws error when i run this as standalone:
![image](https://github.com/user-attachments/assets/5b5580e6-23de-43b5-828b-744c0fcdc803)
should i ignore that and remove it or keep it?

Reply:
Huh, though I updated that comment once I saw that.. or perhaps that was in your other PR? The lint is right, ignore this one.